### PR TITLE
docs(PRT): revise mf6io and supptechinfo

### PR DESCRIPTION
**Pull request by wpbonelli**
Incorporate suggestions after review

### CODE REVIEW

File: doc/mf6io/mf6ivar/dfn/prt-prp.dfn [hunk 1]
```diff
@@ -189,7 +189,7 @@ in_record true
 tagged false
 repeating true
 longname release times
-description times to release, relative to the beginning of the simulation.
+description times to release, relative to the beginning of the simulation.  RELEASE_TIMES and RELEASE_TIMESFILE are mutually exclusive.  Explicit release times and release settings configured in the period block are additive and can be mixed and matched.
```
Comment (langevin-usgs) [inline doc/mf6io/mf6ivar/dfn/prt-prp.dfn:192]: fyi, think you need to escape underscores with a preceding backslash

Comment (wpbonelli) [reply doc/mf6io/mf6ivar/dfn/prt-prp.dfn]: thanks @langevin-usgs

File: doc/SuppTechInfo/particle-tracking-model.tex
Comment (aprovost-usgs) [inline doc/SuppTechInfo/particle-tracking-model.tex:1]: Suggest:
The cell centroid calculated by PRT is not necessarily identical to the cell center specified by the user for a DISV grid.

File: doc/mf6io/mf6ivar/dfn/prt-prp.dfn
Comment (aprovost-usgs) [inline doc/mf6io/mf6ivar/dfn/prt-prp.dfn:1]: Regarding
"Explicit release times and release settings configured in the period block are additive and can be mixed and matched."
(Two instances)  Thinking maybe there's another place to mention this -- discuss?

File: doc/mf6io/mf6ivar/md/mf6ivar.md
Comment (aprovost-usgs) [inline doc/mf6io/mf6ivar/md/mf6ivar.md:1]: auto-generated, so won't comment

File: doc/mf6io/mf6ivar/tex/prt-mip-desc.tex
Comment (aprovost-usgs) [inline doc/mf6io/mf6ivar/tex/prt-mip-desc.tex:1]: auto-generated, so won't comment

File: doc/mf6io/mf6ivar/tex/prt-oc-desc.tex
Comment (aprovost-usgs) [inline doc/mf6io/mf6ivar/tex/prt-oc-desc.tex:1]: auto-generated, so won't comment

File: doc/mf6io/mf6ivar/tex/prt-prp-desc.tex
Comment (aprovost-usgs) [inline doc/mf6io/mf6ivar/tex/prt-prp-desc.tex:1]: auto-generated ...

File: doc/mf6io/prt/prt.tex
Comment (aprovost-usgs) [inline doc/mf6io/prt/prt.tex:1]: Suggest:
To accommodate non-rectangular cells, the face numbering scheme in the PRT Model is based on clockwise ordering of the cell faces.  For a DIS-grid cell, face 1 is the \`\`northern" face, i.e., the face parallel to the $x$ axis that has the greater $y$ coordinate.  For a DISV-grid cell, face 1 is the face that extends in the clockwise direction from the first vertex listed for that cell in the CELL2D input block.
(Note: will want to check the statement about the northern face being #1 to make sure it's true)

**Review by aprovost-usgs — COMMENTED** [review]
Thanks for these fixes.  If there are any other issues from Richard's review the remain to be resolved, we can discuss

File: doc/mf6io/prt/prt.tex
Comment (aprovost-usgs) [reply doc/mf6io/prt/prt.tex:1]: Followup:

checked the IFLOWFACE (PRT) ordering for a DIS-grid cell, and I had it wrong in my review.  Face 1 is the "western" face, i.e., the face parallel to the y axis that has the lesser x coordinate.

also, checked the IFACE (MODPATH 7) ordering, and it's "left, right, front, back, bottom, and top."  Think I had erroneously written "back, front" in the supptechinfo chapter.  can't remember if it also appears in mf6io, though don't think so

File: doc/SuppTechInfo/particle-tracking-model.tex
Comment (aprovost-usgs) [reply doc/SuppTechInfo/particle-tracking-model.tex:1]: Followup:

For the description of the w's in eqns 11-1 and 11-2 in supptechinfo:
After the phrase describing "subscripted v", which ends with "direction, ", insert "and subscripted w (1/T) indicates the constant rate at which a velocity component changes with xtilde or ytilde."  Then start a new sentence to describe xtilde and ytilde: "The coordinates xtilde and ytilde are Cartesian coordinates ..."

**Pull request merged**